doc/man/8/ceph.rst

```diff
@@ -248,6 +248,15 @@ Usage::
 	ceph df {detail}
 
 
+features
```

Comment:
this is the command's name. like `df` and `fs`. so it should be lowercased, i think.